Formula/libpgm.rb

```diff
@@ -19,18 +20,59 @@ class Libpgm < Formula
     sha256 cellar: :any, yosemite:       "ae0d1d980f84677fcaa08b1d9f35f1c9d4858e4239598530b7485e9f248def73"
   end
 
-  # Fix -flat_namespace being used on Big Sur and later.
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
-    sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
-    directory "openpgm/pgm"
-  end
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  # Fix build on ARM
+  patch :DATA
 
   def install
-    cd "openpgm/pgm" do
-      system "./configure", "--disable-dependency-tracking",
-                            "--prefix=#{prefix}"
+    workdir = build.stable? ? "openpgm/pgm" : "pgm"
+    cd workdir do
+      # Fix version number
+      cp "openpgm-5.2.pc.in", "openpgm-5.3.pc.in" if build.stable?
+      system "./bootstrap.sh"
+      system "./configure", *std_configure_args
       system "make", "install"
     end
   end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <pgm/pgm.h>
+
+      int main(void) {
+        pgm_error_t* pgm_err = NULL;
+        if (!pgm_init (&pgm_err)) {
+          return 1;
+        }
+        return 0;
+      }
+    EOS
+    system ENV.cc, "test.c", "-I#{include}/pgm-5.3", "-L#{lib}", "-lpgm", "-o", "test"
```

Comment:
out of my comfort zone with c languages, so I could not understand why I needed to pass both `-L` and `-l`

Reply:
`-L#{lib}` is needed to tell `ENV.cc` where to find the library, while `-lpgm` tells it what library to actually link.